sharktank_models/benchmarks/conftest.py

```diff
@@ -91,6 +97,10 @@ def pytest_sessionfinish(session, exitstatus):
                 )
                 print("\n" + table_data, file=job_summary)
 
+    # on default, if no tests are collected, it exits with error. Instead, return with success status
+    if exitstatus == NO_TESTS_COLLECTED:
+        session.exitstatus = TEST_SUCCESS
```

Comment:
Why? We should avoid having silent failures, and running zero tests (e.g. on a CI workflow) is a failure that we shouldn't ignore.

We go out of our way to avoid the silent failure mode when using ctest: https://github.com/iree-org/iree/blob/5c7c1f2a3c1d3cab7e6c8f659a33fbf164fd929d/build_tools/cmake/ctest_all.sh#L157.